superset-frontend/src/datasource/ChangeDatasourceModal.tsx

```diff
@@ -120,6 +119,21 @@ const ChangeDatasourceModal: FunctionComponent<ChangeDatasourceModalProps> = ({
     setFilter((event.currentTarget?.value as string) ?? '');
   };
 
+  const columns = useMemo(
```

Comment:
It doesn't seem like this `TABLE_COLUMNS` is used anywhere else... should we just define a `const` using this `map` up between lines 36-37, and refer to the `const` here, rather than using `useMemo` at all?

Reply:
Good point! Done